ziti/cmd/create/create_config_controller.go

```diff
@@ -203,7 +203,7 @@ func SetControllerIdentity(data *ControllerTemplateValues) {
 func SetControllerIdentityCert(c *ControllerTemplateValues) {
 	val := os.Getenv(constants.PkiCtrlCertVarName)
 	if val == "" {
-		val = helpers.GetZitiHome() + "/" + helpers.HostnameOrNetworkName() + ".cert" // default
+		val = helpers.GetZitiHome() + "/" + helpers.HostnameOrNetworkName() + "client.chain.cert" // default
```

Comment:
We should consistently name it like *.chain.cert or *.chain.pem. The `pki create client` command uses *.chain.pem to match the pre-existing `pki create server` which also uses *.chain.pem.

ref: https://github.com/openziti/ziti/blob/v1.3.3/ziti/cmd/pki/pki_create_client.go#L181